tidalapi/page.py

```diff
@@ -99,17 +145,30 @@ def get(self, endpoint: str, params: Optional[Dict[str, Any]] = None) -> "Page":
         return self.parse(json_obj)
 
 
-class PageCategory(object):
+@dataclass
+class More:
+    api_path: str
+    title: str
+
+    @staticmethod
+    def from_response(json_obj: JsonObj) -> Optional["More"]:
+        show_more = json_obj.get("showMore")
+        if show_more is None:
+            return None
+        return More(api_path=show_more["apiPath"], title=show_more["title"])
```

Comment:
nit: I prefer an explicit `else:` (one branch per return type is easier to grok imho)

Reply:
Done.